CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Snyk Changelog
 
-## [2.6.1]
+## [2.7.0]
```

Comment:
I'm not sure if I should make this edit, or let gradle do it...

Reply:
Nah, so far we don't write the changelog automatically. Also, you need to release manually, as automatic releases only increase patch versions.